components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap

```diff
@@ -173,41 +173,7 @@ exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
                           class="PodBrowser-root PodBrowser-detailText PodBrowser-body1 PodBrowser-detailText"
                           data-testid="agent-web-id"
                         >
-                          Example B
```

Comment:
this is incorrectly updated - we should see all the agents as we did before this was updated